.github/support.md

```diff
@@ -0,0 +1,12 @@
+# General Discussion
```

Comment:
Where is this `support.md` file linked from?

Reply:
It is supposed to be used as an issue template. See #6